modules/salt/pages/large-scale-tuning.adoc

```diff
@@ -50,6 +50,7 @@ Tuning one parameter might require tuning other parameters, or changing hardware
 When you change a parameter, follow the arrows from that node on the graph to determine what other parameters might need adjustment.
 Continue through each parameter until you have visited all nodes on the graph.
 
+// missing file?
```

Comment:
![image](https://user-images.githubusercontent.com/3914967/67179144-c56a9600-f418-11e9-96e5-0574ff4e7bcb.png)

Nope. Definitely still there.

Reply:
Somehow the link is wrong: https://github.com/SUSE/doc-susemanager/blob/master/modules/salt/pages/parameter_graph.svg

Reply:
no, it won't render in github.